server/acorn

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
```

Comment:
83 files has been added , that's too many changes for a adding a menu

Reply:
i have only worked on 4 files indes.js , index.css , footer.js, footer.css                                              it's actually that packages that me and my team found on our project folder there are 77+ plus of them and no one have an idea how they got installed or by whom anyway might need some help
 git rid of them